go/oasis-test-runner/scenario/e2e/gas_fees_staking.go

```diff
@@ -58,6 +58,20 @@ type gasFeesImpl struct {
 	dumpRestore bool
 }
 
+func (sc *gasFeesImpl) Clone() scenario.Scenario {
+	return &gasFeesImpl{
+		logger: logging.GetLogger("scenario/e2e/gas-fees/staking"),
+	}
+}
+
+func (sc *gasFeesImpl) Name() string {
+	return "gas-fees/staking"
```

Comment:
This seems incorrect, previously the name came from `basicImpl` and was passed to `newBasicImpl` above (see from line ~25 forward). Now the same name will be used for different scenarios.